src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -143,10 +147,10 @@ public String getResourceURI(IResource resource) {
 	@Override
 	public <T extends IResource> List<T> list(String kind, String namespace, Map<String, String> labels) {
 		try {
-			if(!getTypeMappings().containsKey(kind))
+			if(!typeMapper.isSupported(null, kind))
```

Comment:
apparently passing "null" as version has a special meaning. I would document this in javadoc